### PR TITLE
Existing valkyrie feature tests should work without Wings

### PR DESCRIPTION
**Pull request by cjcolvar**
Adjusts the existing feature tests which use valkyrie to run cleanly on a non-wings adapter without making any calling fedora.

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: spec/features/create_work_spec.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
   let(:file2) { File.open(fixture_path + '/image.jp2') }
   let!(:uploaded_file1) { Hyrax::UploadedFile.create(file: file1, user: user) }
   let!(:uploaded_file2) { Hyrax::UploadedFile.create(file: file2, user: user) }
-  let(:permission_template) { create(:permission_template, source_id: 'admin_set/default', with_admin_set: true, with_active_workflow: true) }
+  let(:permission_template) { create(:permission_template, with_admin_set: true, with_active_workflow: true) }
```
Comment (cjcolvar) [inline spec/features/create_work_spec.rb:9]: The source_id was never referenced explicitly so removing it allows it to be any admin set.

File: spec/features/create_work_spec.rb [hunk 2]
```diff
@@ -135,7 +135,7 @@
     end
   end
 
-  context "with valkyrie resources", metadata_adapter: :postgres_adapter do
+  context "with valkyrie resources", valkyrie_adapter: :postgres_adapter do
```
Comment (cjcolvar) [inline spec/features/create_work_spec.rb:138]: `metadata_adapter` was the wrong metadata tag and wasn't actually being handled.

File: spec/features/valkyrie_search_spec.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-RSpec.describe 'searching', index_adapter: :solr_index do
+RSpec.describe 'searching', index_adapter: :solr_index, valkyrie_adapter: :test_adapter do
```
Comment (cjcolvar) [inline spec/features/valkyrie_search_spec.rb:2]: Specifying a `valkyrie_adapter` other than wings ensures that this test works without ActiveFedora.

File: spec/factories/permission_templates.rb [hunk 4]
```diff
@@ -85,4 +76,36 @@ def self.create_access(permission_template_id, agent_type, access, agent_ids)
       end
     end
   end
+
+  class SourceFinder
+    def self.find_or_create_admin_set(source_id)
+      Hyrax.config.use_valkyrie? ? find_or_create_admin_set_valkyrie(source_id) : find_or_create_admin_set_active_fedora(source_id)
```
Comment (dunn) [inline spec/factories/permission_templates.rb:82]: is `use_valkyrie?` deprecated?

Comment (cjcolvar) [reply spec/factories/permission_templates.rb:82]: I'm still seeing it used in a number of places so I don't think so.  I'm open to using a different approach than `use_valkyrie?` here to determine which model to create.  Do you have any suggestions?

Comment (dunn) [reply spec/factories/permission_templates.rb:82]: oh my mistake, i was remembering seeing https://github.com/samvera/hyrax/blob/main/lib/wings/services/custom_queries/find_file_metadata.rb#L110 but that's specifically for wings queries.

**Review by dunn — APPROVED** [review]

**Pull request merged**